docs/changes/time/11680.feature.rst

```diff
@@ -0,0 +1,2 @@
+Adds a new method ``earth_rotation_angle`` to calculate the Local Earth Rotation Angle.
+Also adjusts Local Sidereal Time for ``TIO`` where appropriate.
```

Comment:
This should expand on the implication of adjusting Local Sidereal Time, and define `TIO`. What does this mean for users? Is this an API change that might break downstream tests by giving different values now?

Reply:
Yes, it will. But the change is mainly due to the addition of the TIO locator which is very small. It amounts to a linear offset of ~3 us per century from J2000.

Reply:
The polar offsets add an additional ~+/-50 ns.